src/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+}
+
+.block-stars {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
```

Comment:
as the task says you do not need a fixed width for container "Don't set fixed container size. Let the content size dictate it." As a hint try to get rid of .block-stars